tests/tests/dos.rs

```diff
@@ -245,6 +245,23 @@ async fn complex_queries__41_full_block__query_to_complex() {
     assert!(result.contains("Query is too complex."));
 }
 
+#[tokio::test]
+async fn complex_queries__increased_block_cost__query_too_complex() {
+    let query = FULL_BLOCK_QUERY.to_string();
+    let query = query.replace("$NUMBER_OF_BLOCKS", "1");
+
+    let mut config = Config::local_node();
+    config.graphql_config.costs.block_header *= 41;
+    config.graphql_config.costs.block_transactions *= 41;
+    config.graphql_config.costs.block_header *= 41;
```

Comment:
```suggestion
    config.graphql_config.costs.block_header *= 1000;
```
passes the test

Reply:
Addressed in https://github.com/FuelLabs/fuel-core/pull/2335/commits/58790851f446e19150d1541f850498fe1d0871cd

Changed to use `config.graphql_config.max_queries_complexity` so this test doesn't suddenly fail if that value is modified later as well.